### PR TITLE
Update favicon config

### PR DESCRIPTION
**Pull request by davidyuk**
Generated Firefox app icon

Before 😆  | After
---|---
firefox_app_512x512 | firefox_app_512x512

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -25,22 +25,15 @@ module.exports = {
       .plugin('favicons')
       .use(FaviconsWebpackPlugin, [{
         logo: path.resolve(__dirname, 'src/assets/favicon.svg'),
-        mode: 'webapp', // optional can be 'webapp' or 'light' - 'webapp' by default
-        devMode: 'webapp', // optional can be 'webapp' or 'light' - 'light' by default
```
Comment (davidyuk) [inline vue.config.js:29]: The default values should be fine. I don't think that it is necessary to generate `2732x2048` images on each dev run.

File: vue.config.js [hunk 2]
```diff
@@ -25,22 +25,15 @@ module.exports = {
...
         publicPath: '/',
         favicons: {
           start_url: '/',
-          appName: 'You Do Not Need A Cape To Be A Hero',
+          appName: 'Superhero.com',
           appDescription: 'You Do Not Need A Cape To Be A Hero',
```
Comment (davidyuk) [inline vue.config.js:32]: Probably this is an outdated description.

File: vue.config.js [hunk 3]
```diff
@@ -25,22 +25,15 @@ module.exports = {
...
           developerName: 'Aeternity Developers',
-          developerURL: 'https://github.com/kenodressel/aepp-boilerplate',
-          background: '#ff0d6a',
```
Comment (davidyuk) [inline vue.config.js:37]: The colors from Base aepp 😅

File: vue.config.js [hunk 4]
```diff
@@ -25,22 +25,15 @@ module.exports = {
...
-          theme_color: '#ff0d6a',
-          icons: {
-            coast: false,
-            yandex: false,
-            windows: false,
-          },
```
Comment (davidyuk) [inline vue.config.js:43]: I can't find an explanation of why it is disabled in git blame. I think it is ok to use all features implemented in this package.

**Review by CedrikNikita — APPROVED** [review]

**Pull request merged**